src/FffGcodeWriter.cpp

```diff
@@ -519,10 +519,10 @@ void FffGcodeWriter::processRaft(const SliceDataStorage& storage)
         // When we use raft, we need to make sure that all used extruders for this print will get primed on the first raft layer,
         // and then switch back to the original extruder.
         std::vector<unsigned int> extruder_order = getUsedExtrudersOnLayerExcludingStartingExtruder(storage, extruder_nr, layer_nr);
-        for (unsigned int each_extruder_need_prime : extruder_order)
+        for (unsigned int each_to_be_primed_extruder_nr : extruder_order)
```

Comment:
This is not each number!
This is a for-each loop, but that doesn't mean you should put 'each' in the variable name! That's not relevant to this variable.